pkg/gitreceive/build.go

```diff
@@ -112,6 +110,20 @@ func build(
 		}
 	}
 
+	_, disableCaching := appConf.Values["DEIS_DISABLE_CACHE"]
```

Comment:
iirc, the application is now checking if it's 'set', which would mean this logic is correct. We could still change it though, although having `DEIS_DISABLE_CACHE=0` makes no sense to me and I'd rather see it being unset.